client/src/app/core/repositories/agenda/agenda-item-repository.service.ts

```diff
@@ -120,11 +120,12 @@ export class AgendaItemRepositoryService extends MeetingModelBaseRepository<View
      * @param viewModel the view model that the update is based on
      */
     public async update(update: Partial<AgendaItem>, viewModel: ViewAgendaItem): Promise<void> {
-        (<any>update)._itemNumber = update.item_number;
-        const sendUpdate = viewModel.getUpdatedModelData(update);
-        const clone = JSON.parse(JSON.stringify(sendUpdate));
-        clone.item_number = clone._itemNumber;
-        return await super.update(clone, viewModel);
+        // (<any>update)._itemNumber = update.item_number;
```

Comment:
This is ok ;) The commented-out code was a brutal hack for a bug somewhere in the django rest framework which is still not patched...

Reply:
I leave it as it is, until the actions for the agenda-repo are implemented ;)